db/migrate/20220203052500_create_social_networks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSocialNetworks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :social_networks do |t|
+      t.string :twitter
+      t.string :Facebook
+      t.string :LinkendIn
+      t.string :Instagram
```

Comment:
What is the use of placing the names of social networks here?

Reply:
They are going to save a link to the social network